src/js/components/Video/Video.js

```diff
@@ -182,7 +182,11 @@ const Video = forwardRef(
           const scrubberRect = scrubberRef.current.getBoundingClientRect();
           const percent =
             (event.clientX - scrubberRect.left) / scrubberRect.width;
-          videoRef.current.currentTime = duration * percent;
+          if (!duration)
+            console.warn('The video provided has no valid duration property.');
```

Comment:
Is there a chance that the video hasn't been loaded from the server yet? If so, I don't know that we'd want to add this message into the browser console.

Reply:
Not sure if a server error will land in this space, but in some cases, you might have a malfunctioning video that has an issue with its duration.

Reply:
I wasn't thinking there was a server error, just that the latency or bandwidth was slower. I wouldn't want to show a console warning in that case. I think that's more likely than a malfunctioning video.

Reply:
So, should I change the message to:
"The video provided has no valid duration property, check your server connection."
Or you are suggesting not to have a warning at all?

Reply:
no warning

Reply:
In this case, the next line would break the code and send an error, are we good with it?

Reply:
No, I was thinking we'd do:
```
if (duration) videoRef.current.currentTime = duration * percent;
```

Reply:
Thanks. ready.